q2cli/cli.py

```diff
@@ -25,32 +25,45 @@ class QiimeCLI(click.MultiCommand):
     _builtin_commands = {'tools': q2cli._tools.tools,
                          'info': q2cli._info.info}
 
+    @property
+    def _name_map(self):
+        plugins = list(sorted(self._plugin_manager.plugins))
+        accepted_plugins = []
```

Comment:
Change this to `name_map = {}` and populate it with converted name to original name, then return `name_map` at end of function.